roles/openshift_logging/README.md

```diff
@@ -0,0 +1,72 @@
+## openshift_logging Role
+
+This role is used for installing the Aggregated Logging stack. It should be run against
+a single host, it will create any missing certificates and API objects that the current
+[logging deployer](https://github.com/openshift/origin-aggregated-logging/tree/master/deployer) does.
+
+###Required vars:
+
+- install_logging: True
+
+###Optional vars:
+
+- image_prefix: The prefix for the logging images to use. Defaults to 'docker.io/openshift/origin-'.
+- image_version: The image version for the logging images to use. Defaults to 'latest'.
+- logging_use_ops: If 'True', set up a second ES and Kibana cluster for infrastructure logs. Defaults to 'False'.
+- master_url: The URL for the Kubernetes master, this does not need to be public facing but should be accessible from within the cluster. Defaults to 'https://kubernetes.default.svc.cluster.local'.
+- public_master_url: The public facing URL for the Kubernetes master, this is used for Authentication redirection. Defaults to 'https://localhost:8443'.
+- logging_namespace: The namespace that Aggregated Logging will be installed in. Defaults to 'logging'.
+- etcd_generated_certs_dir: The path to the etcd generated certs, this role will create its certs in the logging directory at this path. Defaults to ''/etc/etcd/generated_certs'.
```

Comment:
I would be agreeable to this change.  We moved it here because of a location suggested by @detiber but its not a logical place IMO.